kitt_bar_app/current_batch.rb

```diff
@@ -40,20 +40,20 @@ def header
   end
 
   def tickets
-	  puts "- Tickets|href=#{tickets_url}|size=12"    
+	  puts "🎟 Tickets|href=#{tickets_url}|size=12"    
   end
 
   def end_ticket
     return unless @ticket
 
     url = "https://kitt.lewagon.com/api/v1/tickets/#{@ticket['id']}/mark_as_solved"
-    puts "- Validate ticket|shell=\"#{__dir__}/ticket_validator.rb\" param1=#{KITT_COOKIE} param2=#{url}"
+    puts "✅ Validate ticket|shell=\"#{__dir__}/ticket_validator.rb\" param1=#{KITT_COOKIE} param2=#{url}"
   end
 
   def day_team
-    return if @api_data['on_duties'].empty?
+    return if @api_data['on_duties']&.empty? || @api_data['on_duties'].nil?
```

Comment:
Do we want to .dig() here too ?

Reply:
Agreed, we should use dig consistently 👍